example/example/static/htmx.js

```diff
@@ -5122,7 +5122,8 @@ var htmx = (function() {
 
 /**
  * @typedef {Object} HtmxExtension
- * @see https://htmx.org/extensions/#defining
+ * For version 1 HTMX: @see https://v1.htmx.org/extensions/#defining
+ * For version 2 HTMX: @see https://htmx.org/extensions/building/
```

Comment:
This is a downloaded copy of htmx, we shouldn't edit it. Maybe a fix is needed upstream instead?

```suggestion
 * @see https://htmx.org/extensions/#defining
```